api/src/main/java/com/viaversion/viaversion/api/ViaManager.java

```diff
@@ -140,6 +140,13 @@ default void setDebug(boolean debug) {
      */
     void addEnableListener(Runnable runnable);
 
+    /**
+     * Adds a runnable to be executed when ViaVersion has finished its init after the full server load.
+     *
+     * @param runnable runnable to be executed
+     */
+    void addLateEnableListener(Runnable runnable);
```

Comment:
PostEnable

Reply:
Renamed, I also removed the javadoc and added @ Override since the methods come from the interface